artemis/src/main/java/tech/pegasys/artemis/cli/deposit/CommonParams.java

```diff
@@ -74,7 +73,7 @@
       paramLabel = "<GWEI>",
       converter = UnsignedLongConverter.class,
       description = "Deposit amount in Gwei")
-  private UnsignedLong amount;
+  private UnsignedLong amount = UnsignedLong.valueOf(32_000_000_000L);
```

Comment:
Is this how you specify a default?

Reply:
Yes, though I've just realised it was deliberately set to not have a default and be required.  That seems reasonable since the deposit amount varies on custom networks so much so I've gone back to that.